oneswap_helper.rb

```diff
@@ -549,13 +625,18 @@ def detect_context_package(distro)
     def context_command(disk, osinfo)
         cmd = nil
         if osinfo['name'] == 'windows'
-            return false
+            context_fullpath = detect_context_package('windows')
+            context_basename = File.basename(context_fullpath)
+            cmd = 'virt-customize -q'\
+                  " -a #{disk}"\
+                  " --copy-in #{context_fullpath}:/Temp"\
+                  " --firstboot-command 'msiexec -i c:\\Temp\\#{context_basename} /quiet && powershell -executionpolicy bypass -File \"C:\\Program Files (x86)\\Encore\\one-context-network\\one_context_networking_reset.ps1\"'"
```

Comment:
idk if we plan on merging this upstream but do we want Encore hard coded in here?

Reply:
Yea i'll take that out before i send it to them. This is just to merge to our internal master